addons/popochiu/engine/objects/graphic_interface/templates/9_verb/9_verb_gui.gd

```diff
@@ -208,4 +225,8 @@ func _on_quit_pressed() -> void:
 	quit_popup.open()
 
 
+func _show_use_in(item: PopochiuInventoryItem) -> void:
+	G.show_hover_text("%s %s in" % [E.get_current_command_name(), item.description])
```

Comment:
Is the right preposition `in` or `on` here?

Reply:
Not sure. My knowledge of English doesn't go that far. But `on` sounds better.

Reply:
Is the meaning "use something with something else"?

In that case it can be "Use <thing> on <thing>" or "Use <thing> with <thing>", but "in" sounds odd to me.

Reply:
Is as you say. I'll change it. Thanks!